similarity-search/src/main/java/org/phenotips/similarity/internal/SolrSimilarPatientsFinder.java

```diff
@@ -204,4 +237,24 @@ private long count(SolrQuery query)
         }
         return 0;
     }
+
+    private void appendGenesToQuery(PatientData<Map<String, String>> allGenes, StringBuilder q)
+    {
+        if (allGenes != null && allGenes.size() > 0 && allGenes.isIndexed()) {
+            for (Map<String, String> gene : allGenes) {
+                String geneName = gene.get("gene");
+                String status = gene.get("status");
+                if ("solved".equals(status) || "candidate".equals(status)) {
```

Comment:
Empty status should count as candidate as well, to ensure this works properly before 1.3M1.